packages/webdriverio/src/commands/element/getValue.ts

```diff
@@ -20,11 +22,13 @@
  * @uses protocol/elements, protocol/elementIdProperty
  *
  */
-export function getValue (this: WebdriverIO.Element) {
+export function getValue<T>(this: WebdriverIO.Element): Promise<T>
```

Comment:
The `getValue` command is meant to receive the value for an input element which is, in the browser case, always a string. Can you describe the native mobile case?

Reply:
This is not true, the command returns the value property which can be anything.

You can verify this by going to https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_input_test
Open the devtools, select one on the input elements and then with the element selected, input the following into the console: `$0.value = 2`

The value property will be a number but the value attribute will be a strin representation of it.

Furthermore this function also works on custom elements that have a value property which again can hold any data, even big objects.

Reply:
This is what I observe:
<img width="395" alt="Screenshot 2025-02-21 at 12 15 31 PM" src="https://github.com/user-attachments/assets/aae43c2d-bf5f-477c-a686-2294f05b64cc" />

So in the browser the `value` will be always a `string`. Even if I change the value to a number:

<img width="131" alt="Screenshot 2025-02-21 at 12 17 15 PM" src="https://github.com/user-attachments/assets/e6197088-4724-4f2a-9274-8fe05a21b55d" />

Note: there are two commands: `getValue` and `getProperty`. The `getValue` command will always return a `string` in the browser while `getProperty` can return any type.